### PR TITLE
Clarify Feed and Subject

### PR DESCRIPTION
**Pull request by SteveLasker**
Explains Feed as the logical collection, with `CWT_Claims sub` (Subject) as the property to represent the collection.

### CODE REVIEW

**Review by fournet — CHANGES_REQUESTED** [review]
For simplicity, I would prefer to keep only one of the two in the Terminology.

**Comment by henkbirkholz** [comment]
> For simplicity, I would prefer to keep only one of the two in the Terminology.

That seems pretty reasonable to me. Are there any reasons to keep both?

**Comment by SteveLasker** [comment]
> > For simplicity, I would prefer to keep only one of the two in the Terminology.
> 
> That seems pretty reasonable to me. Are there any reasons to keep both?

While I'd agree less is more, this keeps coming back when we try to explain it.

- A feed is a known concept for a collection of related data
- We're using CWT_Claims `sub`, which also makes sense to align with existing specs
- We avoided Artifact ID

Attempting to describe a collection of statements as a subject doesn't resonate.

We can keep Feed as the concept, and simply say CWT_Claims `sub` is how you set an identifier for the feed.
How does that work?

**Comment by henkbirkholz** [comment]
Then we should deprecate the term feed in discussions, too, and just explain how a subject is used as a subject in this context?

**Review by henkbirkholz — CHANGES_REQUESTED** [review]
Feed stills seems to be a superfluous term.

**Review by OR13 — CHANGES_REQUESTED** [review]
I will revise my suggestion to something that is mergable

File: draft-ietf-scitt-architecture.md [hunk 1]
```diff
@@ -213,7 +213,9 @@ Equivocation:
 
 Feed:
 
-: see Subject
+: A logical collection of Statements about the same Artifact.
+For any step or set of steps in a supply chain there may be multiple statements made about the same Artifact.
+Issuers use `CWT_Claims:sub` (Subject)  to create a coherent sequence of Signed Statements about the same Artifact and Verifiers use the Feed to ensure completeness and Non-equivocation in supply chain evidence by identifying all Transparent Statements linked to the one(s) they are evaluating.
```
Comment (OR13) [inline draft-ietf-scitt-architecture.md:218]:
```suggestion
: An unordered collection of arbitrary data (artifacts, signed statements, receipts, or other data) related to the text identifiers, observed by the transparency service in the subject field of the CWT claim property of the protected header of Signed Statements.
```

File: draft-ietf-scitt-architecture.md [hunk 2]
```diff
@@ -257,10 +259,8 @@ The Statement is considered opaque to Transparency Service, and MAY be encrypted
 
 Subject:
 
-: a logical collection of Statements about the same Artifact.
-For any step or set of steps in a supply chain there may be multiple statements made about the same Artifact.
-Issuers use Subject to create a coherent sequence of Signed Statements about the same Artifact and Verifiers use the Subject to ensure completeness and Non-equivocation in supply chain evidence by identifying all Transparent Statements linked to the one(s) they are evaluating.
-In SCITT, Subject is a property of the dedicated, protected header attribute `15: CWT_Claims` within the protected header of the COSE envelope.
+: See Feed as a logical collection of Statements about the same Artifact.
+Subject is a property of the dedicated, protected header attribute `15: CWT_Claims` within the protected header of the COSE envelope, to correlate the Statements.
```
Comment (OR13) [inline draft-ietf-scitt-architecture.md:263]:
```suggestion
: This term has the same definition as in RFC8392.
```

**Comment by SteveLasker** [comment]
Updated to merge in PR feedback

**Comment by SteveLasker** [comment]
@henkbirkholz, @OR13, @fournet, PTAL as I've made some updates.

**Pull request merged**